libraries/AP_Mount/AP_Mount_Backend.h

```diff
@@ -337,6 +335,8 @@ class AP_Mount_Backend
         Location poi_loc;         // location of the POI
         Quaternion att_quat;      // attitude quaternion of the gimbal
     } poi_calculation;
+    // calculate the Location that the gimbal is pointing at
+    void calculate_poi();
```

Comment:
maybe also keep calculate_poi() up higher as well.  I totally understand the intention is to move the methods to be with the members that they use but that's now how the rest of the file is organised and I'd rather not mix styles

Reply:
Done